src/components/common/Feedback/Feedback.module.scss

```diff
@@ -0,0 +1,39 @@
+@import "../../../styles/settings.scss";
+
+@keyframes fadeIn {
+  from {
+    opacity: 0;
+  }
+
```

Comment:
Wydaje mi się, że tutaj nie powinno być pustej linii.

Reply:
@MaciekLubocki To jest ok, w tej chwili formatowania i poprawnośći scss pilnuja stylelint i prettier. Dlatego podczas review kwestie formatowanie można pomijać - oczywiście, gdyby wkradł się jakis duży problem bo ktoś edytuje kod z poziomu GH a nie lokalnie to trzeba zareagować. Ale poto to przygotowaliśmy te wszystkie narzędzie, aby takie zadania zdjąć z programistów i delegować je do narzedzi które to zutomatycznie wykonają